docs/readme.txt

```diff
@@ -7,7 +7,7 @@ The official website for Sakai is:
 	
 The latest release of Sakai can always be found at:
 
-	https://www.sakailms.org/release
+	http://source.sakaiproject.org/release/22.2/
```

Comment:
http://source.sakaiproject.org/release/

Reply:
Updated the changes.

Reply:
Will you please merge the PR

Reply:
Thanks!. I don't have permissions but I'll get someone else to do it.

Reply:
Thank you!